docs/evaluate/development-production-features/multi-tenant.mdx

```diff
@@ -0,0 +1,39 @@
+---
+id: multi-tenant
+title: Multi-tenant - Temporal feature
+description: Learn about Temporal Cloud's multi-tenant architecture and how it enhances scalability, efficiency, and cost-effectiveness.
+sidebar_label: Multi-tenant
+tags:
+- multi-tenant
+
+keywords:
+- multi-tenant
+- Temporal Cloud
+- cloud architecture
+- scalability
+- cost-effectiveness
+- noisy neighbor 
+- database performance
+- high throughput
+---
+
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
+Temporal Cloud's architecture scales to handle multiple tenants efficiently.
+Our features enable you to handle multiple Namespaces sharing the same compute and persistence layer, ensuring performance and operational efficiency.
```

Comment:
Sorry for being nitpicky on this, but this is also stylistically strange to me! "Our features" seems marketing-y in this context... I'd rather see this written as "this feature"